src/response.js

```diff
@@ -124,6 +124,25 @@ export default class Response extends Body {
 		return response;
 	}
 
+	static json(data = undefined, init = {}) {
```

Comment:
Not sure if this is in the spec, but Chrome throws different errors if called without `data` vs. explicitly passing `undefined`:

![Screenshot 2022-11-09 at 09 29 56](https://user-images.githubusercontent.com/189580/200778574-7e6e8a7b-ff15-4977-933a-c9f884b8b68a.png)

Not sure we need to follow this though, but I think that the `= undefined` should be removed? It communicates that the argument is optional, but it is in fact required...

```suggestion
	static json(data, init = {}) {
```

Reply:
`any` webidl types mean that the type is optional and Response.json.length should be 0

Reply:
see https://github.com/nodejs/undici/pull/1723

Reply:
That's cool, today I learned!